.github/ISSUE_TEMPLATE/add_org.yml

```diff
@@ -0,0 +1,50 @@
+name: 🌐 Add Orgnaisation
+description: 🌐 Add a new organisation
+title: "🌐 Add new organisation"
+labels: ["Status: triage"]
+body:
+  - type: input
+    attributes:
+      label: Organisation Name
+      description: Name of organisation goes here
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Organisation Logo
+      description: Logo of organisation goes here
+    validations:
+      required: true
+  - type: textarea
```

Comment:
I'll suggest to request just the basic information at this first stage of discussion. Will get the logo later.

Reply:
Sure thing I will do the changes.